src/common/Executor.cpp

```diff
@@ -238,6 +238,10 @@ void Executor::setupSuite()
 
   getCout() << "\nSetting up suite based on input..." << endl;
 
+#if defined(RAJA_ENABLE_SYCL)
+  KernelBase::qu = camp::resources::Sycl().get_queue();
```

Comment:
This should probably be the queue from the default Sycl resource.

Reply:
Yup. This will get straightened out in a subsequent PR. Issue to resolve: https://github.com/LLNL/RAJAPerf/issues/434